client/ayon_core/hosts/maya/api/lib.py

```diff
@@ -1906,12 +1897,15 @@ def assign_look_by_version(nodes, version_id):
     project_name = get_current_project_name()
 
     # Get representations of shader file and relationships
-    look_representation = ayon_api.get_representation_by_name(
-        project_name, "ma", version_id
-    )
-    json_representation = ayon_api.get_representation_by_name(
-        project_name, "json", version_id
+    representations = ayon_api.get_representations(
+        project_name=project_name,
+        representation_names={"ma", "json"},
+        version_ids=[version_id]
     )
+    look_representation = next(
+        repre for repre in representations if repre["name"] == "ma")
+    json_representation = next(
+        repre for repre in representations if repre["name"] == "json")
```

Comment:
what about using lambda here. do you think it would be more clearer ? 
imo, I think it'd be more obvious that it returns the representation that name is `ma`

```suggestion
    look_representation = next(filter(lambda repre: repre["name"] == "ma" , representations))
    json_representation = next(filter(lambda repre: repre["name"] == "json" , representations))
```

Reply:
I'm fine with either actually. This seems like a personal preference, maybe @iLLiCiTiT can weigh in?

Reply:
I like it more as it is. Combination of `next`, `filter` and `lambda` in single line scares me.